src/stream-tcp.c

```diff
@@ -933,11 +933,6 @@ static int StreamTcpPacketStateNone(ThreadVars *tv, Packet *p,
             StatsIncr(tv, stt->counter_tcp_midstream_pickups);
         }
 
-        /* reverse packet and flow */
```

Comment:
Why do these lines get removed ?

By the way, there is one CI failure about S-V test http-sticky on Fedora... to be investigated

Reply:
They're no longer required; in the case that we start a flow on the SYN-ACK the flow is now created in the proper client->server direction, rather then server->client, which required this flipping after the fact.

Reply:
ok thanks